### PR TITLE
Add seeker creation form

### LINKED ISSUE #118: Add Seeker creation form
Add urls, view and html to handle everything from the user side

### PR DESCRIPTION
**Pull request by Micha546**
This PR is depended on #100 !!!
Do not merge this before #100 - Already merged

Adding Seekers creation form, html to show the form,
urls in roo_me.urls and seekers.urls to route to the new view
and view to handle the creation and saving of the new Seeker!
🧙 

Fix #118

### CODE REVIEW

**Review by DanielMy777 — COMMENTED** [review]
Excellent!!!!!!! ✔️ ✔️ ✔️

File: seekers/forms.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            'num_of_rooms',
+            'about',
+        )
+
+    def clean_min_rent(self):
+        min_rent = self.cleaned_data.get('min_rent')
+
+        if not min_rent:
+            raise forms.ValidationError('must have min rent')
+        if min_rent < 0:
```
Comment (simzacks) [inline seekers/forms.py]: see: https://stackoverflow.com/questions/12384460/allow-only-positive-decimal-numbers

File: seekers/forms.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            'num_of_roomates',
+            'num_of_rooms',
+            'about',
+        )
+
+    def clean_min_rent(self):
+        min_rent = self.cleaned_data.get('min_rent')
+
+        if not min_rent:
+            raise forms.ValidationError('must have min rent')
```
Comment (simzacks) [inline seekers/forms.py]: see https://docs.djangoproject.com/en/3.2/ref/forms/fields/
for enforcing required fields without needing to have an if statement for each one

**Review by simzacks — CHANGES_REQUESTED** [review]

File: seekers/tests.py [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+        'num_of_rooms': num_of_rooms,
+        'about': about,
+    })
+
+    assert form.is_valid() is validity
+
+
+@pytest.fixture
+def valid_form(new_city):
+    return SeekerCreationForm(data={
```
Comment (Omeramsc) [inline seekers/tests.py:46]: shouldn't you use `form.is_valid()`?

Comment (Omeramsc) [reply seekers/tests.py:46]: Sorry, I didn't notice It's the fixture itself. 
Do not change it, but do add a separate test that asserts an insertion of a valid form was successful.

Comment (Micha546) [reply seekers/tests.py:46]: Will do ☝️

**Comment by Micha546** [comment]
@simzacks @Omeramsc I've changed the code, 
The clean_max_rent is still there because that's (I think) how you compare 2 form fields:
https://stackoverflow.com/questions/58068844/django-how-to-validate-validator-a-greater-then-between-two-formfields

**Review by Omeramsc — APPROVED** [review]
LGTM

**Review by simzacks — APPROVED** [review]

**Pull request merged**